### DISCUSSION TITLE
2-layer tailings stack runout

### DISCUSSION THREAD

**Discussion by mikael-onsilva** [Anura3D/Anura3D_OpenSource#146] [Q&A]
Hello everyone,

I'm working on this tailings stack pile, where I need to model its runout. The failure will be caused by the liquefaction of the bottom portion of the stack and we need to evaluate what will happen to the upper portion of the stack and how it will all runout to downstream.

Is it posible to model this kind of case in Anura3D?

Thanks for your help.

**Comment by jacobosaa** [comment]
Hi!  
Maybe you could read the column collapse example in the tutorial, asign the material properties for each tailing layer, and i think that it's possible that you could assign to that bottom layer some damping and accelerations properties. 
I'm also working in tailing modelling in Anura3D, maybe we could share some ideas.


Greetings from chile!

**Comment by mikael-onsilva** [comment]
Hi Jacob,

First of all, thanks for the answer.
I'll take a look into the example mentioned. The only thing is that this case is a static liquefaction, but I don't think there is going to be any difference.

Sure, I'll keep you posted on how this modelling goes.

Best regards.
